README.rst

```diff
@@ -1,45 +1,51 @@
-====
-scipion4facilities
-====
+=========================
+Scipion4facilities plugin
+=========================
 
-**scipion4facilities** is a Python module of Scipion framework for cryo-EM facilities
+This plugin allows to use different utils for cryo-EM facilities (like monitors)
+within the Scipion framework.
 
++------------------+------------------+
+| stable: |stable| | devel: | |devel| |
++------------------+------------------+
 
-There is a type of protocols called monitors which are used to produce live analysis plots, generate reports or raise alerts when some problems are detected. A monitor example is the CTF-monitor, that checks the computed defocus values for each micrograph as they are generated. CTF-monitor may raise an alert if the defocus values are above or below certain thresholds. A special case of this monitors is the monitor summary which encapsulates the CTF Monitor, the system monitor and the movie gain monitor and continuosly creates a report.
+.. |stable| image:: http://scipion-test.cnb.csic.es:9980/badges/facilities_prod.svg
+.. |devel| image:: http://scipion-test.cnb.csic.es:9980/badges/facilities_sdevel.svg
 
-This module contains protocols and utilities related with monitors.
+Please, check the `facilities documentation <https://scipion-em.github.io/docs/docs/facilities/facilities.html>`_ for more details.
 
-See https://scipion-em.github.io/docs/release-3.0.0/docs/facilities/customize-report.html for more information
 
--------------
-Development
--------------
+Installation
+------------
 
-To install **scipion4facilities** for development purposes, one can do:
+You will need to use `Scipion 3.0 <https://github.com/I2PC/scipion/releases/tag/V3.0.0>`_
```

Comment:
Thus should point somewhere else: https://scipion-em.github.io/docs/release-3.0.0/index.html ?

Reply:
Right, I've copied from scipion-em. We should change it also there.